src/main/treeDataProviders/utils/dependencyUtils.ts

```diff
@@ -164,6 +402,37 @@ export class DependencyUtils {
         return projectNode.dependenciesWithIssue.length;
     }
 
+    /**
+     * Create an Issue node child for a dependency issues node base on a given IVulnerability
+     * @param issue - the issue to create the child node from
+     * @param dependencyWithIssue - the parent of this issue
+     * @param severity - the severity of this issue
+     * @param impactedPath - the impacted path graph of this issue
+     */
+    private static populateDependencyIssue(
+        issue: IVulnerability,
+        dependencyWithIssue: DependencyIssuesTreeNode,
+        severity: Severity,
+        component: IComponent,
+        impactedPath: IImpactGraph
+    ) {
+        let violationIssue: IViolation = <IViolation>issue;
+        if (violationIssue && violationIssue.license_key) {
+            // License violation
+            dependencyWithIssue.issues.push(new LicenseIssueTreeNode(violationIssue, severity, dependencyWithIssue, impactedPath));
+        } else {
```

Comment:
Let's use early return in this function. if-else-if-else is too much to comprehend for the random reader.